ministryofjustice/correspondence_tool_staff
# CT-2614 fix ping healthcheck fix for CTS

## Description


## Self-review checklist


### Screenshots
Old ping / healthcheck:

![image](https://user-images.githubusercontent.com/22935203/84505760-37e4fa80-acb6-11ea-9aba-313e02bffc9e.png)
![image](https://user-images.githubusercontent.com/22935203/84505699-197eff00-acb6-11ea-808b-cf899b130887.png)


New ping/healthcheck:




![image](https://user-images.githubusercontent.com/22935203/84505829-54813280-acb6-11ea-864a-dc76f831d217.png)
![image](https://user-images.githubusercontent.com/22935203/84505867-6236b800-acb6-11ea-8042-fb0408650ada.png)

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2614
(1 of 2 PRs for this ticket)

### Deployment
normal, have to rebuild and deploy

### Manual testing instructions
Go to /ping and /healthcheck should both make sense and work. Expect Redis Queue to be false as there are dead jobs and retry jobs in queue, but that shouldn't cause 502 by itself.
